fastly.yaml

```diff
@@ -0,0 +1,4 @@
+---
+vhost:
```

Comment:
`vhost` is a term we've inherited from the script, so that's why we kept it. We might be able to do something about that, if the need is there.